src/NUnitFramework/nunitlite.tests/TestUtilities/TestBuilder.cs

```diff
@@ -48,9 +48,7 @@ public static TestSuite MakeFixture(Type type)
 
         public static TestSuite MakeFixture(object fixture)
         {
-            TestSuite suite = MakeFixture(fixture.GetType());
-            suite.Fixture = fixture;
-            return suite;
+            return MakeFixture(fixture.GetType());
```

Comment:
We'll need to look at every use of this method. When a test pre-creates a fixture, it's generally because we have "primed" the instance with certain values for the test. That fixture must be used for the test to succeed. Most likely, we will have to stuff it into the context, if we don't already. (Hart to tell from the narrow point of view of the diff)

This also applies to the following method.